presto-raptor/src/main/java/com/facebook/presto/raptor/metadata/DatabaseShardManager.java

```diff
@@ -102,6 +105,8 @@
 public class DatabaseShardManager
         implements ShardManager
 {
+    private final CounterStat deltaChangedShards = new CounterStat();
+    private final CounterStat deletedShardsForDeltaDelete = new CounterStat();
```

Comment:
You can wrap these 2 counters into a `DeltaDeleteStats` class and rename them as `updatedShards` and `deletedShards`.